### DISCUSSION TITLE
How do we store assorted media like live music DVDs?

### DISCUSSION THREAD

**Discussion by POD-io** [jellyfin/jellyfin#11922] [Q&A] [answered]
I have a lot of live music blu ray downloads, which dont have an IMDB entry. This leads to Jellyfin labeling the media vastly incorrect, typically grabbing the artist name and matching it to a movie title.

How do others deal with this? Can you toggle metadata retrieval off for certain titles, or is there a way to manually add your own and prevent future updates?

Generally, what does everyone do with titles that fall outside of typical movie & TV media?

**Comment by TheMelmacian** [comment, marked as answer]
First you could check if other metadata provider (thetvdb.com, themoviedb.org, omdbapi.com etc.) have an entry for your music bluray/dvd.

If not you can write your own [nfo file](https://jellyfin.org/docs/general/server/metadata/nfo) and save it alongside your media file and jellyfin would read the metadata from this file. [TinyMediaManager](https://www.tinymediamanager.org/) can help you to create nfo files but it is possible that some fields differ from what jellfin expects.

It's also possible to edit the metadata via the Metadata Editor from the GUI (... -> Edit Metadata).
At the bottom of the editor there are also some flags that can be set to block either all metadata from being automatically updated or only certain fields.
